numaprom/_constants.py

```diff
@@ -54,12 +56,29 @@
         "num_epochs": 100,
         "training_keys": ["namespace", "name"],
         "scrape_interval": 5,
+        "unified_anomaly": "namespace_argocd_o11yfuzzygqlfederation_segment_api_unified_anomaly_qal",
         "metrics": [
-            "namespace_http_numalogic_o11yfuzzygqlfederation_segment_api_latency",
-            "namespace_http_numalogic_o11yfuzzygqlfederation_segment_api_error_count",
+            "namespace_argocd_o11yfuzzygqlfederation_segment_api_latency_qal",
```

Comment:
Do you think maybe we can get env from `os.env`? And use an f-string with it here?

Reply:
No, lets not complicate it here. Its any way a temporary solution.